runtime/cli.go

```diff
@@ -0,0 +1,20 @@
+package runtime
+
+import (
+	"cosmossdk.io/depinject"
+	"github.com/spf13/cobra"
+
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+)
+
+// AutoCLIConfig is
+type AutoCLIConfig struct {
+	// AutoCLIOptions
+	AutoCLIOptions     *autocliv1.ModuleOptions
+	CustomQueryCommand *cobra.Command
+	CustomTxCommand    *cobra.Command
+}
+
+func (C AutoCLIConfig) IsOnePerModuleType() {}
```

Comment:
Trying to figure out where this `OnePerModuleType` belongs. Alternatives are:
* keep it here in runtime like this
* keep it in runtime, but instead do something like `type AutoCLIOptions *autocliv1.ModuleOptions` and just use the commands on `AppModuleBasic` instead
* make `*autocliv1.ModuleOptions` a `OnePerModuleType`, although so far we _only_ have generated code in the `api` module
* put it in `client/v2/autocli` but that would introduce a dependency of cosmos-sdk on `client/v2` - I'm not sure which way we want the dependency to go yet

Reply:
After discussing with @amaurym, I'm attempting the extension interface approach (6).